<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminLibrarian</class>
 <widget class="QMainWindow" name="AdminLibrarian">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BookInventory_Button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>160</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Book Inventory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RoomInventory_Button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>160</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Room Inventory</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>240</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inventories</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
